ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -326,6 +345,10 @@ class FamilyReads extends React.PureComponent {
       openFamily && (igvSamplesByFamilySampleIndividual || {})[openFamily]) || {}
     const dnaTrackOptions = DNA_TRACK_TYPE_OPTIONS.filter(({ value }) => igvSampleIndividuals[value])
     const rnaTrackOptions = RNA_TRACK_TYPE_OPTIONS.filter(({ value }) => igvSampleIndividuals[value])
+    const project = openFamily && projectsByGuid[familiesByGuid[openFamily].projectGuid]
+    const geneLocus = project && variant && getGeneLocus(variant, genesById, project)
+    const locusOptions = geneLocus ? [{ value: variant && getVariantLocus(variant, project), text: 'Variant' },
```

Comment:
`geneLocus` is already conditional on `variant`, so theres no need to do `variant && getVariantLocus` here